src/Phpbb/TranslationValidator/Validator/FileValidator.php

```diff
@@ -39,6 +39,81 @@ class FileValidator
 	/** @var \Phpbb\TranslationValidator\Output\OutputInterface */
 	protected $output;
 
+	/** @var array List from https://developers.google.com/recaptcha/docs/language */
+	private $reCaptchaLanguages = [
+		'ar',
+		'af',
+		'am',
+		'hy',
+		'az',
+		'eu',
+		'bn',
+		'bg',
+		'ca',
+		'zh-HK',
+		'zh-CN',
+		'zh-TW',
+		'hr',
+		'cs',
+		'da',
+		'nl',
+		'en-GB',
+		'en',
+		'et',
+		'fil',
+		'fi',
+		'fr',
+		'fr-CA',
+		'gl',
+		'ka',
+		'de',
+		'de-AT',
+		'de-CH',
+		'el',
+		'gu',
+		'iw',
+		'hi',
+		'hu',
+		'is',
+		'id',
+		'it',
+		'ja',
+		'kn',
+		'ko',
+		'lo',
+		'lv',
+		'lt',
+		'ms',
+		'ml',
+		'mr',
+		'mn',
+		'no',
+		'fa',
+		'pl',
+		'pt',
+		'pt-BR',
+		'pt-PT',
+		'ro',
+		'ru',
+		'sr',
+		'si',
+		'sk',
+		'sl',
+		'es',
+		'es-419',
+		'sw',
+		'sv',
+		'ta',
+		'te',
+		'th',
+		'tr',
+		'uk',
+		'ur',
+		'vi',
+		'zu',
+		'', // Allow empty strings
```

Comment:
Why? Without this value, the recaptcha should not work. We ask for a default "en", when nothing fits:

> If no code exists for your language you can use "en" or leave the string empty

Reply:
If no code exists for their language they can use en or leave the string empty. So by having the `''` as part of the array it means validation will still pass if the translator chooses to leave the string empty.

Reply:
Good.